src/applications/vaos/services/appointment/types.js

```diff
@@ -0,0 +1,248 @@
+/**
+ *
+ * @summary
+ * Appointments resources are created from several different data sources:
+ *
+ * - VistA and Video appointments from mobile appointment service (MAS)
+ * - Community care appointments fetched from var-resources service
+ * - VA and CC requests stored fetched from var-resources service
+ *
+ * Records from each of those sources are called appointments, ccAppointments,
+ * and requests, respectively
+ *
+ * @typedef {Object} Appointment
```

Comment:
This isn't fully complete, there are still some variants of the contained/participants array that need to be documented.